src/Notes/WC_Admin_Notes_Update_Version.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * WooCommerce Admin: Update version reminder note.
+ *
+ * Creates a note to nudge users to use the newer version when two are installed.
+ *
+ * @package WooCommerce Admin
+ */
+
+namespace Automattic\WooCommerce\Admin\Notes;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Admin_Notes_Update_Version.
+ */
+class WC_Admin_Notes_Update_Version {
+	const NOTE_NAME = 'wc-admin-update-version';
```

Comment:
Should this be `wc-deactivate-outdated-plugin` to be more reflective of the call to action?

Reply:
Went with `wc-admin-deactivate-plugin`.